src/main/java/com/fauna/event/FeedOptions.java

```diff
@@ -5,97 +5,177 @@
 
 import static com.fauna.constants.Defaults.DEFAULT_TIMEOUT;
 
+/**
+ * Represents the options for configuring a feed in Fauna.
+ * <p>
+ * The {@code FeedOptions} class provides configuration parameters such as cursor,
+ * start timestamp, page size, and timeout for retrieving feeds from Fauna.
+ */
 public class FeedOptions {
+
     private final String cursor;
     private final Long startTs;
     private final Integer pageSize;
     private final Duration timeout;
 
-    public static FeedOptions DEFAULT = FeedOptions.builder().build();
+    /**
+     * The default {@code FeedOptions} instance with default settings.
+     */
+    public static final FeedOptions DEFAULT = FeedOptions.builder().build();
 
-    public FeedOptions(String cursor, Long startTs, Integer pageSize,
-                       Duration timeout) {
+    /**
+     * Constructs a new {@code FeedOptions} with the specified parameters.
+     *
+     * @param cursor   A {@code String} representing the cursor position in the feed. Cannot be provided with a
+     *                 {@code startTs}.
+     * @param startTs  A {@code Long} representing the start timestamp for the feed. Cannot be provided with a
+     *                 {@code cursor}.
```

Comment:
```suggestion
     * @param startTs  A {@code Long} representing the start timestamp for the feed. Represents a time in microseconds since the Unix epoch. Cannot be provided with a
     *                 {@code cursor}.
```